### PR TITLE
Backport PR #24897: DOC: Add ref for every under examples/animation

### PR DESCRIPTION
**Pull request by tacaswell**

### CODE REVIEW

**Comment by tacaswell** [comment]
Manually created this because I jumped the gun on manually asking for a backport and then got both the automatic one and my requested on in flight at the same time and broke meeseeks 😞

File: examples/animation/animate_decay.py [hunk 1]
```diff
@@ -7,6 +7,8 @@
 
 - using a generator to drive an animation,
 - changing axes limits during an animation.
+
+Output generate via `matplotlib.animation.Animation.to_jshtml`.
```
Comment (QuLogic) [inline examples/animation/animate_decay.py:11]:
```suggestion
Output generated via `matplotlib.animation.Animation.to_jshtml`.
```

**Review by QuLogic — COMMENTED** [review]
Ditto for all of them.

Not going to bother with it on `main`, as it'll be fixed by the merge up.

File: examples/animation/animated_histogram.py [hunk 2]
```diff
@@ -45,6 +45,8 @@ def animate(frame_number):
 # ``prepare_animation`` will define ``animate`` function working with supplied
 # `.BarContainer`, all this is used to setup `.FuncAnimation`.
 
+# Output generate via `matplotlib.animation.Animation.to_jshtml`.
```
Comment (oscargus) [inline examples/animation/animated_histogram.py:48]:
```suggestion
# Output generated via `matplotlib.animation.Animation.to_jshtml`.
```

File: examples/animation/animation_demo.py [hunk 3]
```diff
@@ -10,6 +10,8 @@
 examples that use it.
 
 Note that calling `time.sleep` instead of `~.pyplot.pause` would *not* work.
+
+Output generate via `matplotlib.animation.Animation.to_jshtml`.
```
Comment (oscargus) [inline examples/animation/animation_demo.py:14]:
```suggestion
Output generated via `matplotlib.animation.Animation.to_jshtml`.
```

Comment (oscargus) [reply examples/animation/animation_demo.py:14]: And so on...

**Comment by tacaswell** [comment]
Have to replace this because I can not push to this branch because I opened the PR (which makes sense from a security point of view an maintainers can not sneakily get write access to any fork via opening a PR).

**Pull request closed without merging**